packages/cli/lib/exitCodes.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * 0: Successful run
+ * 1: Config problem or internal error
+ * 2: Warnings or validation issues
+ */
+const EXIT_CODES = {
```

Comment:
Would it make sense to put this under `/cli/lib/enums/exitCodes.js`?

Reply:
I can do that 👍